helm/charts/oathkeeper/charts/oathkeeper-master/values.yaml

```diff
@@ -1,3 +1,11 @@
+# Mode for oathkeeper controller
+# Two possible modes are: controller or sidecar
+global:
+  ory:
```

Comment:
Is there a particular reason to nest this that deeply for a single value? Alternatively we could do:

```
global:
  maester:
    mode: controller
```

Reply:
The thing with using global values is that they can be seen well, globally in your installation, as they are exposed by tiller. Because of that, you want to have a detailed tree in order to eliminate confusion and overwriting the wrong values.

Reply:
Ok, that makes sense then!